File: backend/src/main/java/com/app/mevocab/internal/result/dto/request/JoinQuizRequest.java
package com.app.mevocab.internal.result.dto.request;

import lombok.*;
import lombok.experimental.FieldDefaults;

@Builder
@Data
@AllArgsConstructor
@NoArgsConstructor
@FieldDefaults(level = AccessLevel.PRIVATE)
public class JoinQuizRequest {
    String studentID;
    String quizID;
}

File: frontend/lib/data/models/dto/Request/JoinQuizRequest.dart
class JoinQuizRequest {
  final String studentID;
  final String quizID;

  JoinQuizRequest({
    required this.studentID,
    required this.quizID,
  });

  Map<String, dynamic> toJson() {
    return {
      'studentID': studentID,
      'quizID': quizID,
    };
  }
}
